assets/terraform/gce/bootstrap/centos.sh

```diff
@@ -41,8 +50,8 @@ function setup-user {
     chmod 0600 /home/${os_user}/.ssh/authorized_keys
     chown -R $service_uid:$service_gid /home/${os_user}
     # FIXME: make sure that SELinux is in effect for the command below (`getenforce`)
-    semanage fcontext -a -t user_home_t /home/${os_user}
-    restorecon -vR /home/${os_user}
+    retry semanage fcontext -a -t user_home_t /home/${os_user}
```

Comment:
As it turns out, SELinx tooling uses a file system lock and sometimes these ran in parallel with something else that already held the lock - hence I wrapped it into a retry loop.
This should mitigate the SSH connection failures for CentOS/RedHat we were seeing in nightly and PR robotests.